docs/coordinates/frames.rst

```diff
@@ -345,12 +345,19 @@ subclasses/types of transformations are:
     A transform that is defined as a function that takes a frame object
     of one frame class and returns an object of another class.
 
+* `~astropy.coordinates.AffineTransform`
+
+    A transformation that includes a linear matrix operation and a translation
+    (vector offset). These transformations are defined by a 3x3 matrix and a
+    3-vector for the offset. The transformation is applied to the Cartesian
```

Comment:
Maybe add "3-vector for the offset (supplied as a cartesian representation)" -- not very important, though.